action.yml

```diff
@@ -0,0 +1,118 @@
+    description: Prefix to the url address for each of the files(objects)
+    required: false
+  LIFETIME:
+    description: Number of epochs for object to stay valid
+    required: false
+    default: 0
+
+outputs:
+  OUTPUT_CONTAINER_URL:
+    description: Container URL
+    value: ${{ steps.put_files.outputs.container_url }}
+
+runs:
+  using: composite
+  steps:
+    - name: Download latest stable neofs-cli for uploading reports to NeoFS
+      uses: dsaltares/fetch-gh-release-asset@1.1.1
+      with:
+        repo: 'nspcc-dev/neofs-node'
+        version: 'tags/v0.38.1'
+        file: 'neofs-cli-amd64'
+        target: 'neofs/neofs-cli'
+
+    - name: Chmod latest stable neofs-cli
+      shell: bash
+      if: runner.os == 'Linux'
+      run: |
+        sudo chmod a+x neofs-cli
+      working-directory: neofs
+
+    - name: Enable stable neofs-cli
+      shell: bash
+      if: always()
+      run: |
+        echo "$(pwd)" >> $GITHUB_PATH
+      working-directory: neofs
+
+    - name: Create wallet
+      shell: bash
+      env:
+        NEOFS_WALLET: ${{ inputs.NEOFS_WALLET }}
+      run: |
+        echo "$NEOFS_WALLET" | base64 -d > wallet.json
+
+    - name: Define expiration
+      shell: bash
+      env:
+        NEOFS_NETWORK_DOMAIN: ${{ inputs.NEOFS_NETWORK_DOMAIN }}
+        LIFETIME: ${{ inputs.LIFETIME }}
+      run: |
+        if [[ "$LIFETIME" =~ ^[0-9]+$ ]]; then           
+            if [[ "$LIFETIME" -eq "0" ]]; then
+              EXP_EPOCH=0
+            else
+              CURRENT_EPOCH=$(neofs-cli netmap epoch --rpc-endpoint $NEOFS_NETWORK_DOMAIN:8080)
+              EXP_EPOCH=$((LIFETIME + CURRENT_EPOCH))
+            fi
+          else
+            echo "Error: LIFETIME $LIFETIME is not a valid number."
+            exit 1
+          fi
+          echo "EXP_EPOCH=$EXP_EPOCH" >> $GITHUB_ENV
+
+    - name: Prepare venv
+      shell: bash
+      id: prepare_venv
+      run: |
+        python3 -m venv venv
+        source venv/bin/activate && pip install -r requirements.txt
+
+    - name: Put files to NeoFS
+      shell: bash
+      id: put_files
+      env:
+        NEOFS_WALLET_PASSWORD: ${{ inputs.NEOFS_WALLET_PASSWORD }}
+        NEOFS_NETWORK_DOMAIN: ${{ inputs.NEOFS_NETWORK_DOMAIN }}
+        STORE_OBJECTS_CID: ${{ inputs.STORE_OBJECTS_CID }}
+        NEOFS_HTTP_GATE: ${{ inputs.NEOFS_HTTP_GATE }}
+        PATH_TO_FILES_DIR: ${{ inputs.PATH_TO_FILES_DIR }}
+        NEOFS_ATTRIBUTES: ${{ inputs.NEOFS_ATTRIBUTES }}
+        URL_PREFIX: ${{ inputs.URL_PREFIX }}
+      run: |
+        source venv/bin/activate && NEOFS_CLI_PASSWORD=$NEOFS_WALLET_PASSWORD python ./push-to-neofs.py \
+        --expire-at "$EXP_EPOCH" --neofs_domain "$NEOFS_NETWORK_DOMAIN" --attributes "$NEOFS_ATTRIBUTES" \
+        --cid "$STORE_OBJECTS_CID" --files-dir "$PATH_TO_FILES_DIR" --url_path_prefix "$URL_PREFIX" --wallet wallet.json
+        echo "::set-output name=container_url::https://$NEOFS_HTTP_GATE/$STORE_OBJECTS_CID/$URL_PREFIX/"
```

Comment:
`set-output` is deprecated and will be removed soon, can we use output to file instead? https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

It should properly work with the custom action `outputs`.

Reply:
> `set-output` is deprecated and will be removed soon, can we use output to file instead? https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
> 
> It should properly work with the custom action `outputs`.

Fixed